resources/styles/components/questions-dashboard.less

```diff
@@ -23,6 +23,11 @@
     }
   }
 
+  .exercises-display {
+    max-width: @tutor-max-panel-width;
+    margin: 0 auto;
+  }
+
```

Comment:
Hm, is this intentional?  When we choose the `card`  style layout we choose the column layout specifically so it would expand for users who have large monitors so they could be more productive and view more exercises at once.